### PR TITLE
znai initial migration

### PR DESCRIPTION
**Pull request by MykolaGolubyev**

### CODE REVIEW

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+class ZnaiMigrations {
+  @ChangeSet(
+    order = "001",
+    id = "001_add_znai_candidate",
+    author = "MykolaGolubyev"
+  )
+  def migration001(implicit db: MongoDatabase) =
+    Candidate(
+      candidate = "znai",
+      name = "znai",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:16]: This is the "pretty" name, so how about we capitalise this as it appears in the copy of your description?

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      description = "Build functional, maintainable, beautiful User Guides with Znai. Instant pages navigation. " +
+        "Local search. Multiple integrations to work with Python, Java, OpenAPI, etc. " +
+        "Transform \"getting started\" sections into slideshow for your workshops. " +
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:19]: Not sure if the quotes will break rendering, mind if we remove them?

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        "Enterprise Docs Hub.",
```
Comment (marc0der) [inline src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:20]: Not sure I understand the relevance of this last line. Do we need it?

**Review by marc0der — CHANGES_REQUESTED** [review]
Some minor changes to be made, then happy to merge.

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala (hunk 3 above)
Comment (MykolaGolubyev) [reply src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:20]: Happy to re-word. The message I won't to leave by "Enterprise Docs Hub" is that it has support for on-site documentation hub and not just static site generator.

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala (hunk 2 above)
Comment (MykolaGolubyev) [reply src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:19]: will remove.

File: src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala (hunk 1 above)
Comment (MykolaGolubyev) [reply src/main/scala/io/sdkman/changelogs/ZnaiMigrations.scala:16]: we didn't do it for webtau. I can remove capitalization in the text below for consistency.

**Review by marc0der — APPROVED** [review]
LGTM!

**Pull request merged**